### PR TITLE
Don't set default_url_options to a string

### PR DESCRIPTION
**Pull request by gabebw**
Before:

```ruby
config.action_mailer.default_url_options = { host: 'ENV.fetch("HOST")' }
```

After:

```ruby
config.action_mailer.default_url_options = { host: ENV.fetch("HOST") }
```

### CODE REVIEW

File: lib/suspenders/app_builder.rb [hunk 1]
```diff
@@ -235,7 +235,7 @@ def configure_simple_form
     def configure_action_mailer
       action_mailer_host "development", "localhost:#{port}"
       action_mailer_host "test", "www.example.com"
-      action_mailer_host "staging", %{ENV.fetch("HOST")}
+      action_mailer_host "staging", %q{ENV.fetch("HOST")}
```
Comment (gabebw) [inline lib/suspenders/app_builder.rb]: Oops!

File: lib/suspenders/actions.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def replace_in_file(relative_path, find, replace)
     end
 
     def action_mailer_host(rails_env, host)
-      host_config = "config.action_mailer.default_url_options = { host: '#{host}' }"
+      host_config = "config.action_mailer.default_url_options = { host: #{host} }"
```
Comment (gabebw) [inline lib/suspenders/actions.rb]: I think we can leave this as-is?

**Comment by gabebw** [comment]
How can I test that this works?

File: lib/suspenders/actions.rb (hunk 2 above)
Comment (croaky) [inline lib/suspenders/actions.rb]: Could name the variable `config`.

**Comment by croaky** [comment]
Whoops, good call.

**Comment by croaky** [comment]
I think the test failure is because in the test environment, we're now passing in `www.example.com`.

What if we passed in the whole config from `lib/suspenders/app_builder.rb`?

```shell
lib/suspenders/app_builder.rb
236:      action_mailer_host "development", %{{ host: "localhost:#{port}" }'}
237:      action_mailer_host "test", '{ host: "www.example.com" }'
238:      action_mailer_host "staging", '{ host: ENV.fetch("HOST") }'
239:      action_mailer_host "production", '{ host: ENV.fetch("HOST") }'
```

I think the spec could be similar to `spec/features/new_project_spec.rb`'s "generated en.yml is evaluated" spec.

**Comment by croaky** [comment]
Good to squash and merge.

**Pull request merged**